lib/tasks/precompute_gene_ontology.rake

```diff
@@ -0,0 +1,209 @@
+MASTER_OWL_URI = "https://raw.githubusercontent.com/arpcard/aro/master/aro.owl".freeze
+URL_PUBMED = "https://www.ncbi.nlm.nih.gov/pubmed/".freeze
+S3_JSON_BUCKET = "idseq-database".freeze
+S3_ARGANNOT_FASTA = "s3://idseq-database/amr/ARGannot_r2.fasta".freeze
+
+DRUG_CLASSES = {
+  "agly" => ["Aminoglycosides", "0000016"],
+  "bla" => ["Beta-lactamases", "3000001"],
+  "colistin" => ["Colistins", "0000067"],
+  # "fcd" => [], # Unclear which one this refers to
```

Comment:
Is `fcd` a drug class that appears in ARGannot? Could you add a comment about where this one came from?

Reply:
Yes it is. Will do.